webapp/javascript/components/TimelineChart/TimelineChartSelection.ts

```diff
@@ -116,8 +117,14 @@ const handleHeight = 22;
       return isLeftSelecting ? 'left' : isRightSelecting ? 'right' : null;
     }
 
-    function setCursor(type: string) {
-      $('canvas.flot-overlay').css('cursor', type);
+    function cursor() {
```

Comment:
function `cursor`? please add what does this function do to the name. setCursor? changeCursor?